internal/osbuild/assembler_test.go

```diff
@@ -0,0 +1,230 @@
+package osbuild
+
+import (
+	"encoding/json"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"reflect"
+	"testing"
+)
+
+func TestAssembler_UnmarshalJSON(t *testing.T) {
+	type args struct {
+		data []byte
+	}
+	tests := []struct {
+		name      string
+		assembler Assembler
+		args      args
+		wantErr   bool
```

Comment:
I would prefer using a more expressive name like `wantsError` or `shouldError` or even `expectedError` (which will be the actual error type if you want to assert on that).

I don't know what the golang convention is but I don't see a reason for abbreviating variable names when they can be written in full and more expressive way.

Reply:
OK, I changed that to `errorExpected`.